source-google-analytics-data-api/config.yaml

```diff
@@ -0,0 +1,23 @@
+credentials:
```

Comment:
Yes, this error is being caused due to the refresh_token from google oauth2 expiring.

My mock project is always in testing phase, so this will always happens after 7 days.
How do you wish to proceed?

Reply:
Admittedly I'm not familiar with google analytics...how would we get a "persistent" project where the token doesn't expire after 7 days? We're gonna need that in order to have these tests run for any length of time.

Reply:
Thats actually a google Oauth behaviour, it just means that my app is not ""internally"" verified by me as production ready, so my tokens will expire.

If Estuary has a valid mock google app with production ready stuff, i believe that would be the best ( even because we will need for all google related stuff )

Theres a lot of info on how to create a google app, but if you want i can write you a small tutorial

Reply:
New creds were added on the new push